ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContext.scala

```diff
@@ -69,8 +78,8 @@ private[kvutils] trait CommitContext {
       }
 
   /** Get the accessed input key set. */
-  def getAccessedInputKeys: collection.Set[DamlStateKey] =
-    accessedInputKeys
+  def getAccessedInputKeysWithFingerprints: collection.Set[(DamlStateKey, Fingerprint)] =
```

Comment:
We may leave associating fingerprints to accessed input keys to the caller, i.e., that would keep this interface less tailored to pre-execution (as we are using the same fingerprints that have been input this is trivial to do at the caller side) as well as remove the need to have an `inputsWithFingerprints` method.